app/queues/dns/index.server.ts

```diff
@@ -0,0 +1,94 @@
+import { DnsRecordStatus } from '@prisma/client';
+
+import { buildDomain } from '~/utils';
+import { createUserDnsRecord, updateDnsRecordById } from '~/models/dns-record.server';
+import { dnsQueue } from './dns-queue.server';
+
+import type { DnsRecord } from '@prisma/client';
+import type { DnsJobData } from './dns-worker.server';
+
+type Subdomain = { subdomain: string };
+
+function addJob(jobName: string, jobData: DnsJobData) {
+  return dnsQueue.add(jobName, jobData, {
+    attempts: 3,
+    backoff: {
+      type: 'exponential',
+      delay: 60_000,
```

Comment:
I know DNS record update will take maximum 60 seconds. So I assumed most cases it will be less than 60 seconds and used much less number to finish the job as early as possible. I guess this 60 second is intended to avoid any too early retry.

Reply:
I went with 10s, what do you think?

Reply:
NOTE: the way exponential backoff works, it's `2^retry * delay`, so that means 10s, 20s, 80s, 160s, etc.  If we did 10s and 6, it would get to 640s = 10.6 hours.

Reply:
I think 10s with 3 retry reasonable